esp-wifi/build.rs

```diff
@@ -93,6 +93,9 @@ fn main() -> Result<(), Box<dyn Error>> {
     }
 
     // emit config
+    //
+    // keep the defaults aligned with `esp_wifi_sys::include::*` e.g.
```

Comment:
would be good to have a way in `esp-config` to have an expression as a default (since depending on esp-wifi-sys in build-dependencies is not a good thing)

Reply:
Could the build.rs depend on esp-wifi sys and the defaults be derived from there?

Reply:
I tried (not too hard) but esp-wifi-sys wasn't very happy about that: https://github.com/esp-rs/esp-wifi-sys/blob/30d43461335cf5f58317dfc78a5397f48dae2e27/esp-wifi-sys/src/lib.rs#L65-L68

We could work around that but not sure about the bump of the build time 🤔

Reply:
Hmm yeah. What about we move this test logic into esp-wifi-sys instead then? If we only plan to upgrade the blobs once a year or so, could we just check if the config has changed there?

Unless we have other use cases for this, I don't really want to over complicate esp-config.

Reply:
I agree - I like the simplicity of esp-config and I don't see much more use-cases. We can definitely just check what we expect to be the defaults there (or in here in esp-wifi ... it's probably better to do it here since we also have the expected defaults here)

I will add that to this PR then